test/performance/shmem_perf_suite/shmem_bibw_atomics_perf.c

```diff
@@ -281,7 +281,7 @@ static inline void bw_set_metric_info_len(perf_metrics_t * const metric_info)
 {
     atomic_op_type op_type;
 
-    for (op_type = FIRST_OP; op_type < LAST_OP; op_type++) {
+    for (op_type = OP_SET; op_type < LAST_OP; op_type++) {
```

Comment:
Instead of `OP_SET` use `0` or create a `#define FIRST_OP` .

Reply:
You can also write this to document the usage model (C spec says it starts at 0 unless you state otherwise, but this can be nice documentation):
```
typedef enum {
    OP_SET = 0,
    OP_INC,
    ...
```

Reply:
One last nitpick then this is ready to merge.